src/app/file-browser/components/file-list-item/file-list-item.component.scss

```diff
@@ -116,16 +116,31 @@ $bg-transition-length: 0.075s;
 .name {
   @include file-list-col-name;
 
+  &-expanded{
+    height: fit-content !important;
+  }
+
   > div {
     display: flex;
     line-height: 1.25;
     max-width: 100%;
 
-    span {
-      flex: 1 1 auto;
+    .article-name {
```

Comment:
Is `article-name` the best name for this class? I feel like something like `item-name` might make more sense, though ultimately it doesn't matter since this classname is only used here.

Reply:
i think you are right!